test/server/chat.js

```diff
@@ -88,5 +88,17 @@ describe('Chat', function () {
 			Chat.formatText(`:)`, true),
 			`:)`
 		);
+		assert.equal(
+			Chat.formatText(`a\nb\nc`),
+			`a\nb\nc`
+		);
+		assert.equal(
```

Comment:
These should all be `strictEqual` but that is for another PR

Reply:
I'm aware

Reply:
NO THEY SHOULDN'T

Notice the top of this file:

```js
const assert = require('assert').strict;
```

TypeScript marking these as wrong is a TypeScript bug.

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/assert.d.ts

lol, this was fixed 3 days ago.

Reply:
It's not typescript marking these wrong, `strict` still works but is is deprecated

Reply:
What do you mean?

Reply:
`require('assert').strict` is not only not deprecated, `require('assert')` _is_ deprecated...

Reply:
Oh, sorry, I didn't know that.